src/app/app.component.ts

```diff
@@ -6,5 +6,5 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'app';
+  title = 'Track Forever';
```

Comment:
This convention of `title = '...'` followed by `<h1>{{title}}<\h1>` is just something left over from the starting example code to show how [template syntax](https://angular.io/guide/template-syntax) works. You can keep doing this if you want, but it's also fine to just hardcode the title in the html because it never changes.